tools/build_node.js

```diff
@@ -41,10 +41,28 @@ async function buildCJSIndex(name, languages) {
 }
 
 async function buildNodeLanguage(language, options) {
+  const EMIT = `function emitWarning() {
+    if (!emitWarning.warned) {
+      emitWarning.warned = true;
+      process.emitWarning(
+        'Using file extension in specifier is deprecated, use "highlight.js/lib/languages/%%%%" instead'
+      );
```

Comment:
We might want to add a `type` parameter:

> The following additional handling is implemented if the warning `type` is `'DeprecationWarning'`:
>
> - If the `--throw-deprecation` command-line flag is used, the deprecation warning is thrown as an exception rather than being emitted as an event.
> - If the `--no-deprecation` command-line flag is used, the deprecation warning is suppressed.
> - If the `--trace-deprecation` command-line flag is used, the deprecation warning is printed to stderr along with the full stack trace.

```suggestion
      process.emitWarning(
        'Using file extension in specifier is deprecated, use "highlight.js/lib/languages/%%%%" instead',
        'DeprecationWarning'
      );
```